frontend/src/NavigationPath.tsx

```diff
@@ -151,12 +151,19 @@ export enum NavigationPath {
   createPolicyAutomation = '/multicloud/governance/policyautomation/create/:namespace/:name',
   editPolicyAutomation = '/multicloud/governance/policyautomation/edit/:namespace/:name',
   policyDetails = '/multicloud/governance/policies/details/:namespace/:name',
+  discoveredByCluster = '/multicloud/governance/discovered-by-cluster/:kind/:name/',
```

Comment:
I'd like to make the URL paths all build off of each other.

How about this:
* /multicloud/governance/discovered
* /multicloud/governance/discovered/:apiGroup?/:apiVersion/:kind/:policyName
* /multicloud/governance/discovered/:apiGroup?/:apiVersion/:kind/:policyName/?:policyNamespace/:clusterName/detail
* /multicloud/governance/discovered/:apiGroup?/:apiVersion/:kind/:policyName/?:policyNamespace/:clusterName/yaml

Reply:
Which one is optional policyName or policyNamespace?  question mark should be end of param
 Ref:https://reactrouter.com/en/main/route/route#optional-segments

Reply:
Also which one is replacement of   `discoveredByCluster`

Reply:
I think `discoveredByCluster` would be:
* /multicloud/governance/discovered/:apiGroup?/:apiVersion/:kind/:policyName/:policyNamespace?

Reply:
We could drop the namespace for now in the route for discoveredByCluster.

Reply:
Is policyNamespace == cluster?

Reply:
Almost always, but we will be allowing it to be `open-cluster-management-policies` as well. So the namespace doesn't affect the grouping but it affects the search query.